docs/hub/datasets-pandas.md

```diff
@@ -22,26 +57,163 @@ Finally, you can use [Hugging Face paths](/docs/huggingface_hub/guides/hf_file_s
 ```python
 import pandas as pd
 
-df.to_parquet("hf://datasets/username/my_dataset/data.parquet")
+df.to_parquet("hf://datasets/username/my_dataset/imdb.parquet")
 
 # or write in separate files if the dataset has train/validation/test splits
 df_train.to_parquet("hf://datasets/username/my_dataset/train.parquet")
 df_valid.to_parquet("hf://datasets/username/my_dataset/validation.parquet")
 df_test .to_parquet("hf://datasets/username/my_dataset/test.parquet")
 ```
 
-This creates a dataset repository `username/my_dataset` containing your Pandas dataset in Parquet format.
-You can reload it later:
+## Use Images
+
+You can load a folder with a metadata file containing a field for the names or paths to the images, structured like this:
+
+```
+Example 1:            Example 2:
+folder/               folder/
+├── metadata.csv      ├── metadata.csv
+├── img000.png        └── images
+├── img001.png            ├── img000.png
+...                       ...
+└── imgNNN.png            └── imgNNN.png
+```
+
+You can iterate on the images paths like this:
+
+```python
+import pandas as pd
+
+folder_path = "path/to/folder/"
+df = pd.read_csv(folder_path + "metadata.csv")
+for image_path in (folder_path + df["file_name"]):
+    ...
+```
+
+Since the dataset is in a supported structure ("metadata.csv" file with "file_name" field), you can save this dataset to Hugging Face and the Dataset Viewer shows both the metadata and images on Hugging Face.
```

Comment:
```suggestion
Since the dataset is in a supported structure (a `metadata.csv` file with a `file_name` field), you can save this dataset to Hugging Face and the Dataset Viewer shows both the metadata and images on Hugging Face.
```